src/generic/generic-component-page-header/generic-component-page-header.twig

```diff
@@ -1,67 +1,74 @@
 {#
-  Accepts the following context variables:
-  - variant: (string) Could be: default, basic, highlight
+  Parameters:
+    - "identity" (string) (default: ''): name of the site, i.e. "Digital single market"
+    - "breadcrumb" (array)  (default: []): collection of @ecl/generic-component-link
+    - "title" (string) (default: ''): page title
+    - "introduction" (string) (default: '')
+    - "metas" (array) (default: []): array of strings ; each one corresponds to a meta item
```

Comment:
`... to a meta item of @ecl/generic-component-meta` maybe to make it a bit more concrete